.devcontainer/onCreateCommand.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+cd .install
+./install_script.sh
+./install_boost.sh 1.83.0
+cd ../
+./.install/test_deps/install_rust.sh
```

Comment:
rust is a test dependency because it's currently only needed by json?

Reply:
removed as rust is now integral in our build